src/win/fsatracedll.c

```diff
@@ -1,8 +1,11 @@
 #include <windows.h>
+#include <psapi.h>
+#include <stdio.h>
```

Comment:
I guess stdio.h isn't needed?

Reply:
Yep, probably for printf tracing I removed

Reply:
Fixed